src/ocean/util/app/ext/ConfigExt.d

```diff
@@ -419,6 +419,9 @@ class ConfigExt : IApplicationExtension, IArgumentsExtExtension
         if (key.length == 0)
             return "Empty key for config option override: " ~ opt;
 
+        if (!this.config.exists(category, key))
+            return "Attempt to override non-existent config entry";
```

Comment:
Does this also error if loose parsing is enabled ? IMO it should not, but I don't see any code to check for it.

Reply:
Yes, it will. Based on our last discussion with @gavin-norman-sociomantic & co intent is to get rid of loose parsing completely, so it is hardly a problem on its own.

Reply:
My feeling is that arg overriding of config values is a separate question to how the config parser works (i.e. strict or "loose").